data/strings/strings.txt

```diff
@@ -14889,46 +14889,89 @@
 
   [editor_other_info]
     tags = android,ios
-    en = Send a note to OSM editors
-    af = Stuur ’n nota aan OSM-wysigers
-    ar = ارسل ملاحظة إلى محرري خريطة الشارع المفتوحة
-    be = Адправіць нататку рэдактарам OSM
-    bg = Изпращане на бележка до редакторите на OSM
-    ca = Envia una ota als editors d'OSM
-    cs = Odeslat poznámku editorům OSM
-    da = Send en besked til OSM-redaktører
-    de = Notiz an OSM-Redakteure senden
-    el = Σημείωμα στους συντάκτες του OSM
-    es = Enviar nota a los editores de OSM
-    es-MX = Enviar nota a los editores de OSM
-    et = Saada märkus OSM muutjatele
-    eu = Bidali oharra OSM editoreei
-    fa = ارسال یادداشت به ویرایش‌کنندگان OSM
-    fi = Lähetä muistilappu OSM-toimittajille
-    fr = Informer les éditeurs OSM
-    hu = Küldjön jegyzetet az OSM editornak
-    id = Kirim catatan ke editor OSM
-    it = Invia un messaggio a OSM
-    ja = OSMエディタにメモを送信
-    ko = OSM 편집인들에게 쪽지 보내기
-    lt = Siųsti pastabą OSM redaktoriams
-    mr = OpenStreetMap संपादकांना एक चिठ्ठी पाठवा
-    nb = Send et notat til OSM-redaktørene
-    nl = Notitie sturen naar de OSM-editors
-    pl = Poinformuj redaktorów OSM
-    pt = Enviar nota aos editores OSM
-    pt-BR = Envie uma observação para os editores do OSM
-    ro = Trimite un mesaj către editorii OSM
-    ru = Отправить заметку редакторам OSM
-    sk = Odoslať poznámku OSM editorom
-    sv = Skicka en not till OSM-redaktörer
-    sw = Tuma ujumbe kwenye vihariri vya OSM
-    th = ส่งข้อความไปยังตัวปรับแต่ง OSM
-    tr = OSM editörlerine bir not gönderin
-    uk = Надіслати нотатку редакторам OSM
-    vi = Gửi ghi chú cho ban biên tập OSM
-    zh-Hans = 给OSM编辑器发送标记
-    zh-Hant = 給OSM編輯器發送標記
+    en = Open a discussion note
```

Comment:
?
```
    en = Open a discussion note at OpenStreetMap.org
```

Reply:
I think with the description below `OpenStreetMap.org` would be rather redundant